libraries/botframework-streaming/src/protocolAdapter.ts

```diff
@@ -47,22 +50,27 @@ export class ProtocolAdapter {
     }
 
     /// <summary>
-    /// Sends a request over the attached request manager.
+    ///
     /// </summary>
     /// <param name="request">The outgoing request to send.</param>
     /// <param name="cancellationToken">Optional cancellation token.</param>

```

Comment:
The whole comment block should be removed?

Reply:
Yea, the comment block should be removed. Thank you, I missed that.